BREAKING CHANGE: The alert role has been removed for Alert component #DS-1175 (#1398)
lmc-eu/spirit-design-system: docs/migrations/web-twig/MIGRATION-v3.md

Either install newer version of the `spirit-icons` package or add `danger` named icon to your project's icons.

### Alert: `role="alert"`

The `role="alert"` has been removed for the default setting of the component.

The alert role is used to communicate an important and usually time-sensitive message to the user. When this role is added to an element,
the browser will send out an accessible alert event to assistive technology products which can then notify the user.
The alert role should only be used for information that requires the user's immediate attention, which is typically
content that is dynamically displayed (such as form validation message etc.), not for content that appears on page load. It should not be used on HTML that the user hasn't interacted with.

> For more information see [ARIA: alert role][alert-role-documentation].

Reviewer: you probably missed to add these links :)

Either install newer version of the `spirit-icons` package or add `danger` named icon to your project's icons.

> ### Alert: `role="alert"`

Reviewer: Please add this section to the TOC at the top of the file

Either install newer version of the `spirit-icons` package or add `danger` named icon to your project's icons.

### Alert: `role="alert"`

The `role="alert"` has been removed from the default setting of the component.

The alert role is used to communicate an important and usually time-sensitive message to the user. When this role is added to an element,
the browser will send out an accessible alert event to assistive technology products which can then notify the user.
The alert role should only be used for information that requires the user's immediate attention, which is typically
content that is dynamically displayed (such as form validation message etc.), not for content that appears on page load.
It should not be used on HTML that the user hasn't interacted with.

For more information see [ARIA: alert role][alert-role-documentation].

#### Migration Guide

> In cases where you need to use the component as an information that requires the user's immediate attention,

Reviewer 2: ```suggestion
In case you need to use the component as an information that requires the user's immediate attention,
```

> you can use `role="alert"` as an [additional attributes][readme-additional-attributes].

Reviewer 2: ```suggestion
you can use `role="alert"` as an [additional attribute][readme-additional-attributes].
```

Please refer back to this guide or reach out to our team if you encounter any issues during migration.

[alert-role-documentation]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role
[dictionary-placement]: https://github.com/lmc-eu/spirit-design-system/blob/main/docs/DICTIONARIES.md#placement
[dropdown-readme]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-twig/src/Resources/components/Dropdown/README.md

> [readme-additional-attributes]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/README.md#additional-attributes

Reviewer: 😬 please link the web-twig readme, not the web-react one.

Reviewer: ```suggestion
[readme-additional-attributes]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-twig/README.md#additional-attributes
```
